src/scenic/simulators/carla/model.scenic

```diff
@@ -54,6 +54,7 @@ param sun_altitude_angle = Range(-90, 90)
 
 class CarlaActor(DrivingObject):
     carlaActor: None
+    rolename: None
```

Comment:
Would be good to put a comment here explaining how this property is used.

Reply:
Added a comment